daemon/config.go

```diff
@@ -14,42 +13,34 @@ const (
 	disableNetworkBridge = "none"
 )
 
-// Config define the configuration of a docker daemon
-// These are the configuration settings that you pass
-// to the docker daemon when you launch it with say: `docker -d -e lxc`
-// FIXME: separate runtime configuration from http api configuration
-type Config struct {
-	Bridge bridge.Config
-
-	Pidfile              string
-	Root                 string
-	AutoRestart          bool
-	Dns                  []string
-	DnsSearch            []string
-	GraphDriver          string
-	GraphOptions         []string
-	ExecDriver           string
-	ExecOptions          []string
```

Comment:
The Windows native execDriver might not accept any option today, but shouldn't we keep `ExecOptions` in `CommonConfig` nevertheless?